tests/src/main.rs

```diff
@@ -20,9 +20,22 @@ struct Opt {
     /// Only run tests that match these filters
     #[structopt(name = "FILTER")]
     filters: Vec<String>,
+
+    /// The environment to compile to the SPIR-V tests.
+    #[structopt(long)]
+    target_env: Option<String>,
```

Comment:
No structopt shorthand for comma-separated? Also, it might make more sense to always add new flags before `filters`, so that it matches the order they show up in the usage, but that's really just my pedantry.